client/src/pages/Signup/index.tsx

```diff
@@ -3,7 +3,7 @@ import {
   Box, Step, Stepper, StepLabel, Button,
 } from '@mui/material';
 import './style.css';
-import { FirstStep, Signup } from '../../components';
+import { FirstStep, Signup, ThirdStepSignUp } from '../../components';
```

Comment:
```suggestion
import { FirstStep, Signup, ThirdStepSignUp } from '../../components';
```
I think we can give it more descriptive name